src/Lecture1.hs

```diff
@@ -108,7 +114,8 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: [Char] -> Integer
+strSum str = sum(map read (words str) :: [Integer])
```

Comment:
You don't need explicit type annotation here. Since you already specified the expected type as `Integer`, GHC will infer the correct type of `read` as well

```suggestion
strSum str = sum (map read (words str))
```